example/.ocamlformat

```diff
@@ -0,0 +1,10 @@
+version=0.18.0
+wrap-fun-args=false
+let-binding-spacing=compact
+field-space=loose
+sequence-style=separator
+doc-comments=before
+margin=80
+module-item-spacing=sparse
+parens-tuple=always
+parens-tuple-patterns=always
```

Comment:
Isn't `exp-grouping=preserve` missing? (don't forget to reformat from `master` after applying this change)

Reply:
Done

Reply:
I don't have the permission to resolve my own threads, but now it's ok for me :+1: